### PR TITLE
Core update#v0.2.0

### PR DESCRIPTION
**Pull request by frankiebee**
This pr brings sdk up to core version 0.2.0 (currently rc) Do not merge until we have stable tag 

## Testing

- [x] Unit tests added/updated
- [x] Integration tests added/updated
- [ ] Manual testing performed

### CODE REVIEW

File: tests/transfer.test.ts [hunk 1]
```diff
@@ -143,12 +133,14 @@ test('Transfer', async (t) => {
...
-    await run('transfer funds (small top-up)', sendMoney(amount))
-  }
+  // NOTE: Leaving this test as commented for now, not sure if still needed now 
+  // that small values can be sent whenever
+  // {
+  //   const amount = BigInt(10 * 10 ** DECIMAL_PLACES)
+  //   // NOTE: once an account is funded, we can send small amounts!
+  //   const sender = charlie.keyring.accounts.registration.pair
+  //   await run('transfer funds (small top-up)', sendMoney(amount))
+  // }
```
Comment (frankiebee) [inline tests/transfer.test.ts:143]:
```suggestion
```

either delete or find the new minimum

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,12 +10,14 @@ The format extends [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
 Version header format: `[version] Name - year-month-day (entropy-core compatibility: version [range])`
 
-## [UNRELEASED]
+## [UNRELEASED] (entropy-core compatibility: 0.2.0)
 
 ### Added
 
 ### Fixed
 - hot fix for issue #380: try in reverse order for testnet environment
+- `entropy.signingManager.getTimeStamp` -> `entropy.signingManager.getBlockNumber` signature request no longer take a time stamp and now take a block number.
```
Comment (rh0delta) [inline CHANGELOG.md:19]:
```suggestion
- `entropy.signingManager.getTimeStamp` -> `entropy.signingManager.getBlockNumber` signature request no longer takes a time stamp and now takes a block number.
```

File: tests/transfer.test.ts (hunk 1 above)
Comment (rh0delta) [inline tests/transfer.test.ts:143]:
```suggestion
```

File: tests/transfer.test.ts [hunk 3]
```diff
@@ -143,12 +133,14 @@ test('Transfer', async (t) => {
   }
 
   /* Test small top-up */
```
Comment (rh0delta) [inline tests/transfer.test.ts:135]:
```suggestion
```

**Review by rh0delta — DISMISSED** [review]

**Comment by frankiebee** [comment]
DO NOT MERGE UNTILL WE HAVE STABLE RELEASE TAG

File: dev/bin/ENTROPY_CORE_VERSION.sh [hunk 4]
```diff
@@ -1,3 +1,3 @@
 #! /usr/bin/env bash
 
-export ENTROPY_CORE_VERSION=release-v0.1.0
+export ENTROPY_CORE_VERSION=release-v0.2.0-rc.1
```
Comment (frankiebee) [inline dev/bin/ENTROPY_CORE_VERSION.sh:3]: SHOULD BE V0.2.0
```suggestion
export ENTROPY_CORE_VERSION=release-v0.2.0
```

**Review by rh0delta — APPROVED** [review]

File: dev/bin/ENTROPY_CORE_VERSION.sh (hunk 4 above)
Comment (frankiebee) [reply dev/bin/ENTROPY_CORE_VERSION.sh]: https://github.com/entropyxyz/entropy-core/pull/930#pullrequestreview-2172859164

**Pull request merged**